### PR TITLE
festival bug fixes

### PR DESCRIPTION
**Pull request by fritzschoff**
ref #2648

### CODE REVIEW

File: libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.html [hunk 1]
```diff
@@ -128,9 +128,12 @@
...
           (click)="submitMovie(element)">
           <mat-icon svgIcon="archipel_content"></mat-icon>
-          <span>Submit to AC</span>
+          <span>Submit to Archipel Content</span>
         </button>
-        <button mat-icon-button color="primary" [routerLink]="['/c/o/dashboard/titles/', element.movie.id ]" target='_blank'><mat-icon svgIcon="launch"></mat-icon></button>
+        <!-- TODO #2648 for deploy festival v0 -->
+<!--         <button mat-icon-button color="primary" [routerLink]="['/c/o/dashboard/titles/', element.movie.id ]" target='_blank'>
+          <mat-icon svgIcon="review"></mat-icon>
+        </button> -->
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.html:136]: what is going to do this button ?

Comment (fritzschoff) [reply libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.html:136]: will lead you to detail view of the movie, if imported, but there are different routes on the catalog and festival app so right now, just commented out

File: libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.ts [hunk 2]
```diff
@@ -118,6 +126,14 @@ export class TableExtractedMoviesComponent implements OnInit {
    */
   private async addMovie(importState: MovieImportState): Promise<boolean> {
     const data = this.rows.data;
+    const appName = getCurrentApp(this.routerQuery)
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.ts:129]: You should do something generic with the list of all apps : 
```typescript
const apps = ['festival', 'catalog'];
for (const app of apps) {
  importState.movie.main.storeConfig.appAccess[app] = app === appName;
}
```

File: libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.html [hunk 3]
```diff
@@ -31,7 +31,7 @@
       (click)="submitSelectedMovies()">
       <ng-container *ngIf="submittedTitles === 0 else submitinprogress">
         <mat-icon svgIcon="archipel_content"></mat-icon>
-        <span>Submit to Archipel Content</span>
+        <span>{{ renderButtonText() }}</span>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.html:34]: as discussed with vincent write "Publish to Marketplace"

File: apps/festival/festival/src/app/marketplace/wishlist/wishlist.component.ts [hunk 4]
```diff
@@ -28,24 +29,26 @@ export class WishlistComponent implements OnInit {
...
 
   ngOnInit() {
-    this.dataSource$ = this.catalogCartQuery.wishlistWithMovies$.pipe(
+    this.sub = this.catalogCartQuery.wishlistWithMovies$.pipe(
       map(wishlist => wishlist.find(wish => wish.status === 'pending')),
+      tap(wishlist => this.hasWishlist = !!wishlist.movieIds.length),
       filter(wishlist => !!wishlist?.movies?.length),
-      map(wishlist => new MatTableDataSource(wishlist.movies))
-    );
+      tap(wishlist => this.dataSource = new MatTableDataSource(wishlist.movies)),
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply apps/festival/festival/src/app/marketplace/wishlist/wishlist.component.ts:48]: dirty issues dirty code, quite simple

File: libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.ts (hunk 2 above)
Comment (phpgeekfr) [reply libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.ts:129]: You can also use the app type in `libs/utils/src/lib/apps.ts`

Comment (phpgeekfr) [reply libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.ts:129]: @GrandSchtroumpf you want to allow a new movie on both app? This is not depending of the app you currently are for importing ?

Comment (phpgeekfr) [reply libs/movie/src/lib/movie/components/import/table-extracted-elements/movies/movies.component.ts:129]: If the point is to initialize all value, this should have already be done in `createMovieAppAccess`

File: apps/festival/festival/src/app/dashboard/tunnel/movie-tunnel.component.html [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <tunnel-layout [steps]="steps" [exitRedirect]="exitRoute">
   <header fxLayout="row" fxLayoutAlign="space-between center">
     <tunnel-step-stat></tunnel-step-stat>
-    <img asset="logo_archipel_content_primary.svg" type="logo" alt="Archipel Content Logo" />
+    <img asset="archipel_market.png" alt="Archipel Content Logo" />
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply apps/festival/festival/src/app/dashboard/tunnel/movie-tunnel.component.html:4]: nope

**Pull request merged**